src/version/sagas/feedsListSagas.js

```diff
@@ -0,0 +1,23 @@
+import { takeEvery, put } from 'redux-saga/effects';
+
+import { token, domainName } from "../../core/token";
```

Comment:
I'd read domainName from window.config

Reply:
Done.

Domain name is defined in window.config.

**domainName** from "../../core/token" is result of function, which extract domain name from config variables.